data_psi.go

```diff
@@ -23,24 +23,27 @@ const (
 	PSITableTypeTOT     = "TOT"
 	PSITableTypeUnknown = "Unknown"
 )
+
+type PSITableTypeId uint16
```

Comment:
Could you rename this to `PSITableTypeID` ? (with `ID` in capital letters)

Reply:
Sure. I've done the same with `PESIsVideoStreamId` also